vanilla/invalid/java/org/spongepowered/server/SpongeVanilla.java

```diff
@@ -179,9 +174,11 @@ public void onServerStopping() {
     }
 
     public void onServerStopped() throws IOException {
-        SpongeCommon.postState(GameState.SERVER_STOPPED, SpongeEventFactory.createGameStoppedServerEvent(Sponge.getCauseStackManager().getCurrentCause()));
-        ((SqlServiceImpl) this.game.getServiceManager().provideUnchecked(SqlService.class)).close();
-        SpongeCommon.getConfigSaveManager().flush();
+        SpongeImpl.postState(GameState.SERVER_STOPPED, SpongeEventFactory.createGameStoppedServerEvent(Sponge.getCauseStackManager().getCurrentCause()));
```

Comment:
You reverted back to SpongeImpl.

Reply:
Given GameState is also gone, that line might as well go.